### PR TITLE
webdav: FileSystem.Create() returns FileInfo and is used to set PUT r…

### PR DESCRIPTION
**Pull request by DeepDiver1975**
…esponse headers

### CODE REVIEW

**Comment by emersion** [comment]
LGTM apart from this!

File: server.go [hunk 1]
```diff
@@ -194,11 +194,21 @@ func (b *backend) PropPatch(r *http.Request, update *internal.PropertyUpdate) (*
...
+	}
+	if !fi.ModTime.IsZero() {
+		w.Header().Set("Last-Modified", fi.ModTime.UTC().Format(http.TimeFormat))
+	}
+	if fi.ETag != "" {
+		w.Header().Set("ETag", internal.ETag(fi.ETag).String())
+	}
 	w.WriteHeader(http.StatusCreated)
 	// TODO: Last-Modified, ETag, Content-Type if the request has been copied
 	// verbatim
```
[the remark this answers is not in the document]
Comment (DeepDiver1975) [reply server.go]: taken care of ...

File: server.go [hunk 2]
```diff
@@ -194,13 +194,22 @@ func (b *backend) PropPatch(r *http.Request, update *internal.PropertyUpdate) (*
...
+	}
+	if !fi.ModTime.IsZero() {
+		w.Header().Set("Last-Modified", fi.ModTime.UTC().Format(http.TimeFormat))
+	}
+	if fi.ETag != "" {
+		w.Header().Set("ETag", internal.ETag(fi.ETag).String())
+	}
 	w.WriteHeader(http.StatusCreated)
-	// TODO: Last-Modified, ETag, Content-Type if the request has been copied
 	// verbatim
```
Comment (emersion) [inline server.go:213]: Sorry, but I'm going to bother you again: "verbatim" is part of the sentence but isn't been deleted :P

Comment (DeepDiver1975) [reply server.go]: :see_no_evil: brain-off coding is not a good thing ...

**Review by emersion — APPROVED** [review]
Thanks!

**Pull request merged**